### PR TITLE
Update plugins and pipe plugins documentation

### PR DESCRIPTION
**Pull request by ScreamZ**
hi

Accurate the documentation because it did not specify this entry about hooks, and about es6 syntax.

regards :)

### CODE REVIEW

File: source/includes/plugins/_04-creating-plugins.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## Creating plugins
 
+**Note :** You can use the ES2015 (es6) syntax while writing plugins.
```
Comment (benoitvidis) [inline source/includes/plugins/_04-creating-plugins.md]: Depends on the version of node the user runs the Kuzzle stack on.

For the time being, Kuzzle enforces node 4.x version, which partial only ES2015 support.

File: source/includes/plugins/_04-creating-plugins.md [hunk 2]
```diff
@@ -59,7 +61,9 @@ Plugin configuration example:
 
 ### > Pipe plugins
 
-`pipe` plugins, like `listener` plugins, are attached to Kuzzle events, and they are supplied with these events data.  
+`pipe` plugins, like `listener` plugins, are attached to Kuzzle events, and they are supplied with these events data.
```
Comment (benoitvidis) [inline source/includes/plugins/_04-creating-plugins.md]: I would rephrase it with something like

`pipe` plugins, like `listener` plugins, are attached to Kuzzle events, and they are supplied with these events data.

Where a `listener` exposes a `hooks` property, a pipe plugin needs to expose its events in a `pipes` property.

File: source/includes/plugins/_04-creating-plugins.md (hunk 1 above)
Comment (ScreamZ) [inline source/includes/plugins/_04-creating-plugins.md]: I also detected that even when using `"use strict";` statement, node 4 still doesn't support the ES6 import syntax. Therefore we can remove this statement.

**Comment by scottinet** [comment]
Obsoleted by https://github.com/kuzzleio/kuzzle-guide/pull/76
